docs/serving/using-auto-tls.md

```diff
@@ -154,6 +154,10 @@ and which DNS provider validates those requests.
 
 If you choose to use the mode of provisioning certificate per namespace, you need to install `networking-ns-cert` components.
 
+**Note that provisioning certificate per namespace only works with DNS-01
+ challenge. If you choose to use HTTP-01 challenge, you must NOT install this
```

Comment:
Just curious, ns-cert for HTTP-01 is under development? Or it will never support HTTP-01?

Reply:
It will never support HTTP01 as the namespace-level cert is wildcard cert which can only works with DNS01.

Reply:
I see. Thank you.